.github/workflows/helm-lint-test.yml

```
@@ -0,0 +1,64 @@
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.13
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)